### DISCUSSION TITLE
Pinning projects and/or multiple repos as one.

### DISCUSSION THREAD

**Discussion by MaximFedarau** [community/community#33631] [Other Feature Feedback, Questions, & Ideas]
<h1 align="center">🖖 Hello!</h1>

## 🧐 I have a question.
For example, I have a very big project, which consists of web application ( `backend` & `frontend` ), mobile apps ( `Android` & `iOS` ) and a desktop app (maybe, written in `Electron`, for example).
So I wanna show this giant app to everyone on my profile, i. e. pinning. <b>But the only way to do so is to pin all 5 repos!</b> I think, that this is kinda inconvenient.

## 🔍 So:
- Is there any way to pin a public `GitHub Project` with general description & other stuff for this big app.
- Or, maybe, I miss something, and I can demonstrate this kind of applications in other way?

**Comment by cosmicqbit** [comment]
Here is how I would achieve that.

- Create a new repo preferably with the name of project as a title.
- Create a Readme in it and may be include a license as well.
- In the Readme add links to all repos which are part of this main project. 

For eg.

# Project Title

## Frontend
![HTML]()
![CSS]()
![Javascript]()
![Rect]()

## Backend
![Nodejs]()
![MongoDB]()
![Ruby on Rails]()

Hope this helps.

**Reply by MaximFedarau** [reply]
### Yeah, I think this can be considered as a solution.
But this way can be used only for the most valuable projects, because we create another one repo.

**Reply by cosmicqbit** [reply]
Yep this is the ideal approach most of the developers have been sticking to this far. There is no other way around it at the moment. If it helps consider marking this as answered. Thank you ❤️
